components/SectionHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styles from '../styles/components/SectionHeader.module.scss';
+
+type Location = 'justify-start' | 'justify-end';
+
+interface Props {
+  title: string;
+  backgroundColor?: string;
+  foregroundColor?: string;
```

Comment:
can we do what we did with the buttons here and just have a style prop which is an enum of prebuilt styles, this way we can verify accessibility.

Reply:
Fixed